{{ cookiecutter.app_name }}/pyproject.toml

```diff
@@ -154,7 +158,9 @@ system_requires = [
 system_runtime_requires = [
 {%- if cookiecutter.gui_framework == "Toga" %}
```

Comment:
If we're interested in getting rid of canberra modules errors, I was able to do so by installing `libcanberra`.

I'd expect the `libcanberra-gtk3-module` package to do the same on debian.

Reply:
I've made this change; there's an overlap with the standalone PR here, so whichever lands first, we need to make sure we get consistent dependencies.